File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_1.vb
    Public Event SelectionChanged(ByVal sender As System.Object, ByVal e As System.EventArgs)

    Private selectedType As Microsoft.Office.Interop.Excel.XlChartType = _
        Microsoft.Office.Interop.Excel.XlChartType.xlColumnClustered

    Public Property Selection() As Microsoft.Office.Interop.Excel.XlChartType
        Get
            Return Me.selectedType
        End Get
        Set(ByVal value As Microsoft.Office.Interop.Excel.XlChartType)
            Me.selectedType = value
        End Set
    End Property

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--updating-a-chart-in-a-worksheet-using-radio-buttons_1.cs
        public event EventHandler SelectionChanged;

        private Microsoft.Office.Interop.Excel.XlChartType selectedType =
            Microsoft.Office.Interop.Excel.XlChartType.xlColumnClustered;

        public Microsoft.Office.Interop.Excel.XlChartType Selection
        {
            get
            {
                return this.selectedType;
            }
            set
            {
                this.selectedType = value;
            }
        }